doc/subsystems/networking/net-app-api.rst

```diff
@@ -0,0 +1,201 @@
+.. _net_app_api:
+
+Network Application API
+#######################
+
+The Network Application API allows applications to
+
+* **Initialize** the application for networking use. This means for example
+  that if the application needs to have an IPv4 address, and if DHCPv4 is
+  enabled, then the net-app API will make sure that the device will get an
+  IPv4 address before the application is started.
+
+* **Set** various options for the networking subsystem. This means that if the
+  user has set options like IP addresses, IEEE 802.15.4 channel etc. in the
```

Comment:
"channel, etc."

Should this be "the IEEE 802.15.4 channel"? Or perhaps "IEEE 802.15.4 channels"? Not sure, just asking.